contrib/standalone-installer/assets/header.py

```diff
@@ -203,11 +203,82 @@ def install_flatpak (directory, verbose, allow_reinstall, allow_older, uninstall
             print(cmd)
         subprocess.run (cmd)
 
+# Check which package to use
+# - return False to use packaged version
+# - return True for snap/flatpak
+def use_included_version (minimum_version):
+    try:
+        import apt
+    except ModuleNotFoundError:
+        return True
+    cache = apt.Cache()
+    pkg = cache.get ("fwupd")
```

Comment:
does pylint like that?

Reply:
Fixed the pylint complaints about this specific stuff (and a few others in the new code)